lib/resty/etcd/v3.lua

```diff
@@ -679,7 +680,6 @@ local function request_chunk(self, method, path, opts, timeout)
         body    = body,
         query   = query,
         headers = headers,
-        ssl_server_name = self.sni,
```

Comment:
Why we don't need SNI in the retry?

Reply:
this is not retry, it is request after connect.